### ISSUE TITLE
Disabled biomes still generating

### ISSUE THREAD

**Issue by scalda** [Team-RTG/Realistic-Terrain-Generation#267]
Hi

I am still getting biome's generating that are disabled and i can't work out why, I have mangrove and marsh disabled.

using latest release with fresh config files that i matched up the weights and such to previous files

Regards

Scalda

**Comment by whichonespink44** [comment]
@scalda would you be able to zip up your config/mods/scripts folders (and the world save that the biomes are generating in) and upload it to Dropbox or similar? I really need to see this bug in action for myself as I've so far been unable to reproduce it, so if I had your exact environment, I could just replace RTG with a debug version and get to the bottom of it quite quickly.

**Comment by scalda** [comment]
Ok I will create you multimc instance with it all in

**Comment by scalda** [comment]
Here you go

https://drive.google.com/folderview?id=0B41DtbSYPlY9R3ZidUo3b3NEYzg&usp=sharing

**Comment by whichonespink44** [comment]
Never used MultiMC before... why do I have a feeling this is going to change my world completely :wink:  (thanks for uploading, I'm downloading now and will let you know when I've got it)

**Comment by scalda** [comment]
multimc is the best launcher i have found i use it for every modpack i play

**Comment by whichonespink44** [comment]
Cool, looking forward to trying it out! I've downloaded the ZIP file so feel free to delete it from your GDrive if you want.

I'll let you know if I figure anything out.

**Comment by scalda** [comment]
ok thank you

**Comment by scalda** [comment]
Here is a screen shot with co-ords of a mangrove biome, that has just generated in the world you have

**Comment by whichonespink44** [comment]
@scalda I don't think GitHub attached the image to the issue - would you mind posting it again?

**Comment by scalda** [comment]
no it didn't let me try again https://dl.dropboxusercontent.com/u/8871364/Captures/Minecraft%20Screen%20shots/2015-11-05_19.07.40.png

**Comment by whichonespink44** [comment]
Thanks for the screenshot - unfortunately, I wasn't able to load up your world as it has more mods than my laptop can handle. So I created a test world with just RTG & BOP and a couple other mods (along with your config files), and I've been flying around and still haven't found a biome that's been disabled in the configs.

I'm also starting to wonder if it's a problem with just the Mangrove biome. Have you had any other biomes generate when they've been disabled in the configs?

**Comment by scalda** [comment]
i had the marsh biome spawn as well which is disabled in BOP

**Comment by scalda** [comment]
i have just loaded a new world and loaded straight into a mangrove biome

**Comment by whichonespink44** [comment]
Been working on this on and off for the last couple of weeks, and I haven't made any progress as I'm not able to reproduce the problem. Disabled biomes just aren't generating for me.

Have you made any progress with this @scalda ?

I'm going to close this for now, but I'll re-open it if other people start having this issue.

In the meantime, if you can provide a MultMC instance with as few mods as possible, which also includes a world in which the disabled biomes are generating, I'll replace RTG with my debug version and I should be able to figure out what's going on.

**Comment by whichonespink44** [comment]
@scalda Are you still having this issue? If so, I just came across a post that might explain (somewhat) what's happening: http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1282126-extrabiomesxl-universal-3-16-3-for-mc-1-7-10?comment=14924 (Sorry for digging up this old issue, but as soon as I read that post, I immediately thought of this and thought I'd share in case it was helpful.)

**Comment by Ezoteric** [comment]
I still have this issue with bops flower field. :p

**Comment by scalda** [comment]
Hi i have had the problem but i'm still using a old version for Survival
Stories 3 as Iskall85 doesn't want to change it since the pack has launched

**Comment by whichonespink44** [comment]
@Ezoteric Subtract 128 from the Flower Field biome ID... if the result is the same as an existing biome ID, try changing the Flower Field ID to something else (and then obviously make sure that the new ID minus 128 doesn't equal an existing ID, etc). Seems to be an issue specific to BOP, but it might actually be a problem with the way MC handles those mutated biomes. :disappointed: 

@scalda Yeah, not a good idea to change the biome IDs of an existing world (as I'm sure you know), but something to keep in mind in the future. By the way, thanks for including RTG in your Survival Stories mod pack... that explains why YouTube kept putting it in my 'Recommended for You' feed!! :grinning:

**Comment by Ezoteric** [comment]
@whichonespink44 do you suspect i have a conflict or do you know that ? :P

**Comment by whichonespink44** [comment]
@Ezoteric I suspect a conflict, yes... there's just no other explanation for a biome being generated in RTG. It can only generated biomes that have been given an ID, so if it's generating in an RTG world, then it must've gotten through the biome registration process.

Have you tried it yet? Let me know if it works and I'll post something in the OP in case other people experience the issue.

**Comment by Ezoteric** [comment]
Euh.. made dump and flower field ain't even on the list lolz.

**Comment by whichonespink44** [comment]
Did you get the ID from the configs? If it's 58 in the configs, then if the theory is correct, there should be a biome with ID 186 somewhere in the configs as well. If there's no biome with ID 186, then not sure what's going on.

**Comment by Ezoteric** [comment]
yes there is a Biome with ID 186 :p

**Comment by Ezoteric** [comment]
Hmmz, fun fact deciduousForest is ID56, is false, does not generate and i do have a biome on ID 184.

So in your logic this one should also generate right ?

**Comment by whichonespink44** [comment]
Something like that :stuck_out_tongue_winking_eye:

**Comment by Ezoteric** [comment]
You crazy 😂😂
